aevien/master-server-toolkit
# Add MongoDB Module Connection

As you know, MongoDB is one of the most reliable and fast database. Used by a lot of developers for MMO and Server development. 

Think about that, I've created a new Module for MST with the lastest version of MongoDB c# library.

With this, MTS can be connected in LiteDB and MongoDB.

I believe this can be very helpful for a lot of game developers and can make MST more and more reliable and powerful

I made a lot of tests and everything is working perfectly.

This solution dont remove LiteDB, it's just a new Database option to use.

### How to test it

1. Install MongoDB on your machine/Server 
[https://docs.mongodb.com/manual/installation/](https://docs.mongodb.com/manual/installation/)

2. Open any Unity MasterServer scene.
3. Disable GameObject inside MasterServer GameObject called DatabaseFactoryModule 
4. Create a new one and puts MongoDBFactoryModule Component
5. Start your MasterServer (Build or in Editor)

_If you have installed mongodb correctly everything will work fine_
_In case of error of connection. Check if your mongodb service on your machine is running_
